src/pages/docs.tsx

```diff
@@ -165,9 +171,395 @@ function renderArticleSections(
+          uninstall, switch version, etc. To install nvm, use this&nbsp;
+          <a href="https://github.com/nvm-sh/nvm#install--update-script">
+            install script
+          </a>
+          .
+        </p>
+        <p className="t-body1">
+          On Unix / OS X systems Node.js built from source can be installed
+          using&nbsp;<a href="https://github.com/creationix/nvm">nvm</a>&nbsp;by
+          installing into the location that nvm expects:
+        </p>
+        <ShellBox
+          textToCopy={
+            'env VERSION=`python tools/getnodeversion.py` make install DESTDIR=`nvm_version_path v$VERSION` PREFIX=""'
+          }
+        >
+          <span className="function">env</span>
+          &nbsp;VERSION=`python tools/getnodeversion.py`
+          <span className="function">&nbsp;make install&nbsp;</span>
+          DESTDIR=`nvm_version_path v$VERSION` PREFIX=&quot;&quot;
+        </ShellBox>
+        <p className="t-body1">
+          After this you can use <code>nvm</code> to switch between released
+          versions and versions built from source. For example, if the version
+          of Node.js is v8.0.0-pre:
+        </p>
+        <ShellBox textToCopy="nvm use 8">
+          nvm use&nbsp;
+          <span className="function">8</span>
+        </ShellBox>
+        <p className="t-body1">
+          Once the official release is out you will want to uninstall the
+          version built from source:
+        </p>
+        <ShellBox textToCopy="nvm uninstall 8">
+          nvm uninstall&nbsp;
+          <span className="function">8</span>
+        </ShellBox>
+        <h3>OpenBSD</h3>
+        <p className="t-body1">
+          Node.js is available through the ports system.
+        </p>
+        <ShellBox textToCopy="/usr/ports/lang/node">
+          /usr/ports/lang/node
+        </ShellBox>
+        <p className="t-body1">
+          Using&nbsp;
+          <a href="https://man.openbsd.org/OpenBSD-current/man1/pkg_add.1">
+            pkg_add
+          </a>
+          &nbsp;on OpenBSD:
+        </p>
+        <ShellBox textToCopy="pkg_add node">pkg_add node</ShellBox>
+        <h3>openSUSE and SLE</h3>
+        <p className="t-body1">
+          Node.js is available in the main repositories under the following
+          packages:
+        </p>
+        <ul>
+          <li>
+            <p className="t-body1">
+              openSUSE Leap 42.2:&nbsp;
+              <span>
+                <code>nodejs4</code>
+              </span>
+            </p>
+          </li>
+          <li>
+            <p className="t-body1">
+              openSUSE Leap 42.3:&nbsp;
+              <span>
+                <code>nodejs4</code>,&nbsp;<code>nodejs6</code>
+              </span>
+            </p>
+          </li>
+          <li>
+            <p className="t-body1">
+              openSUSE Tumbleweed:&nbsp;
+              <span>
+                <code>nodejs4</code>,&nbsp;<code>nodejs6</code>,&nbsp;
```

Comment:
Lets change node4 and 8 to something more recent...

Reply:
changed to nodejs10, nodejs12, nodejs14  in f257538